### PR TITLE
[8.0][ADD] pos_invoice_journal module

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: pos_invoice_journal/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Pos Invoice Journal
+===================
+
+This module allow you to use a different journal for the invoices created
```
Comment (pedrobaeza) [inline pos_invoice_journal/README.rst]: s/allow/allows

File: pos_invoice_journal/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Pos Invoice Journal',
+    'summary': """
```
Comment (pedrobaeza) [inline pos_invoice_journal/__openerp__.py]: Don't use `"""` for summary. Use string concatenation instead.

File: pos_invoice_journal/static/description/icon.svg [hunk 3]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (pedrobaeza) [inline pos_invoice_journal/static/description/icon.svg:1]: Isn't this as PNG?

File: pos_invoice_journal/README.rst (hunk 1 above)
Comment (tafaRU) [inline pos_invoice_journal/README.rst]: Ok, so also https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst needs to be fixed, isn'it?

File: pos_invoice_journal/static/description/icon.svg (hunk 3 above)
Comment (tafaRU) [inline pos_invoice_journal/static/description/icon.svg:1]: I don't care about this as I used https://github.com/acsone/bobtemplates.odoo/blob/master/bobtemplates/odoo/addon/%2Baddon.name%2B/static/description/icon.svg.oca cc @sbidoul

File: pos_invoice_journal/__openerp__.py (hunk 2 above)
Comment (tafaRU) [inline pos_invoice_journal/__openerp__.py]: Here, too, I used https://github.com/acsone/bobtemplates.odoo/blob/master/bobtemplates/odoo/addon/%2Baddon.name%2B/__openerp__.py.bob#L8-L9

@sbidoul would you like to open an issue on https://github.com/acsone/bobtemplates.odoo ? By the way, thanks a lot for this tool, very helpful!

File: pos_invoice_journal/README.rst (hunk 1 above)
Comment (pedrobaeza) [inline pos_invoice_journal/README.rst]: OK, fixed, but not putting an `s`, but including the word `to`

**Comment by tafaRU** [comment]
@pedrobaeza, I addressed your comments.
Thanks!

File: pos_invoice_journal/README.rst (hunk 1 above)
Comment (tafaRU) [inline pos_invoice_journal/README.rst]: Ok, thanks!

**Comment by legalsylvain** [comment]
:+1: Thanks.

**Comment by rafaelbn** [comment]
Tested in runbot :+1:

**Pull request merged**